spec/controllers/catalog_controller_spec.rb

```diff
@@ -39,30 +39,30 @@
       expect(email).to be_nil
     end
   end
-  describe '#online_holding_note?' do
-    subject(:note) { described_class.new.online_holding_note?(nil, document) }
+  describe '#show_location_has?' do
+    subject(:note) { described_class.new.show_location_has?(nil, document) }
 
     let(:link) { 'field not blank' }
     let(:holdings) do
       '{"1":{"location_has":["blah"]}}'
     end
 
     describe 'for document with link and holding note' do
-      let(:document) { { electronic_access_1display: link, holdings_1display: holdings } }
+      let(:document) { SolrDocument.new({ electronic_access_1display: link, holdings_1display: holdings }) }
```

Comment:
Use a `SolrDocument` in this test rather than a hash, since it is more realistic.